src/core/vector/qgsvectorlayerrenderer.cpp

```diff
@@ -188,7 +188,7 @@ void QgsVectorLayerRenderer::setLayerRenderingTimeHint( int time )
 
 QgsFeedback *QgsVectorLayerRenderer::feedback() const
 {
-  return mFeedback.get();
+  return renderContext()->feedback();
```

Comment:
Can you explain why these changes are necessary? https://github.com/qgis/QGIS/blob/master/src/core/maprenderer/qgsmaprendererjob.cpp#L600 should take care of it already...

Reply:
This is a remain of a test: we did not understand why the feedback was created in the `QgsVectorLayerRenderer` then set back to the job context instead of using the one of the job context. This looked like an inversion of responsibility. But later, we saw some renderer classes use an extended feedback class... 

I will revert this modifition